integration/kubernetes/confidential/sev.bats

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env bats
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export TEST_DIR
+export ENCRYPTION_KEY
+
+export KBS_DIR
+export KBS_DB_HOST
+export KBS_DB_USER="kbsuser"
+export KBS_DB_PW="kbspassword"
+export KBS_DB="simple_kbs"
+export KBS_DB_TYPE="mysql"
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+TESTS_REPO_DIR=$(realpath "$BATS_TEST_DIRNAME/../../..")
+test_tag="[cc][kubernetes][containerd][sev]"
+
+esudo() {
+  sudo -E PATH=$PATH "$@"
+}
+
+run_kbs() {
+  KBS_DIR="$(mktemp -d /tmp/kbs.XXXXXXXX)"
+  pushd "${KBS_DIR}"
+
+  # Clone and run
+  git clone https://github.com/confidential-containers/simple-kbs.git --branch main
+  (cd simple-kbs && git checkout -b branch_0.1.1 0.1.1)
+  (cd simple-kbs && esudo docker-compose up -d)
+  sleep 5
+
+  # Set KBS_DB_HOST to kbs db container IP
+  KBS_DB_HOST=$(esudo docker network inspect simple-kbs_default \
+    | jq -r '.[].Containers[] | select(.Name | test("simple-kbs_db.*")).IPv4Address' \
+    | sed "s|/.*$||g")
+
+  popd
+}
+
+pull_encrypted_image_and_set_keys() {
+  # Pull encrypted docker image - test workload
+  esudo docker pull quay.io/kata-containers/encrypted-image-tests:encrypted
+
+  # Get encryption key from docker image label
+  ENCRYPTION_KEY=$(esudo docker inspect quay.io/kata-containers/encrypted-image-tests:encrypted \
```

Comment:
Clever idea to put encryption and ssh keys attached to labels so we don't need to hard-code on the test code!